geometry_msgs/Point[] face
geometry_msgs/Point[] left_hand
geometry_msgs/Point[] right_hand
geometry_msgs/Point[] pose